plugins/forc-index/src/commands/remove.rs

```diff
@@ -3,11 +3,11 @@ use anyhow::Result;
 use clap::Parser;
 use std::path::PathBuf;
 
-/// Stop and remove a running index.
+/// Stop and remove a running indexer.
 #[derive(Debug, Parser)]
 pub struct Command {
-    /// URL at which index is deployed.
-    #[clap(long, default_value = defaults::INDEXER_SERVICE_HOST, help = "URL at which index is deployed.")]
+    /// URL at which indexer is deployed.
+    #[clap(long, default_value = defaults::INDEXER_SERVICE_HOST, help = "URL at which indexer is deployed.")]
     pub url: String,
 
     /// Path to the manifest of indexer project being deployed.
```

Comment:
Should this also be changed to

> Path to the manifest of _the_ indexer project being deployed

to mirror the change in the revert operation?